src/components/planning/Settings.tsx

```diff
@@ -42,208 +43,78 @@ const ButtonWrap = styled.div`
-            </DateInputWrap>
-          </Div>
-
-          <Hr />
-
-          <h1>Openingsredenen</h1>
-          <ColumnWrap>
-            <Checkboxes options={ allProjects } state={ checkedProjects! } onChangeHOF={ projectsOnChangeHOF } />
-          </ColumnWrap>
-
-          <Hr />
-
-          <h1>Stadia</h1>
-          <ColumnWrap>
-            { lists.map(list => {
-                const {
-                  name,
-                  primaryStadium,
-                  secondaryStadia,
-                  excludeStadia
-                } = list
-                return (
-                  <Div key={ name }>
-                    <h3>{ name }</h3>
-                    <H4>1. Zoveel mogelijk</H4>
-                    <StadiaSelect selected={ primaryStadium[0] } onChange={ primaryStadium[1] } />
-                    <H4>2. Aanvullen met</H4>
-                    <StyledCheckboxes name={ `${ name }-secondary` } options={ stadia! } state={ secondaryStadia[0] } onChangeHOF={ secondaryStadia[1] } />
-                    <H4>3. Uitsluiten</H4>
-                    <StyledCheckboxes name={ `${ name }-exclude` } options={ stadia! } state={ excludeStadia[0] } onChangeHOF={ excludeStadia[1] } />
-                  </Div>
-                )
-              })
-            }
-          </ColumnWrap>
-
-          <ButtonWrap>
-            { showErrorMessage &&
-              <ErrorMessage text={ errorMessage! } />
-            }
-            { showUpdatingSpinner &&
-              <SmallSpinner />
-            }
-            <Button variant="secondary" disabled={ disabled }>Bewaren</Button>
-          </ButtonWrap>
-        </form>
-      }
-
-      { showJSON &&
-        <>
-          <Hr />
-          <JSONDisplay json={ settings } />
-        </>
-      }
+  return isFetching
+  ? <Spinner />
+  : <Wrap>
+        <Form
+          onSubmit={onSubmit}
+          initialValues={data?.settings}
+          render={({ handleSubmit, values, hasValidationErrors }) => (<>
+            <form onSubmit={handleSubmit}>
+              <h1>Peildatum</h1>
+              <Div>
+                <DateInputWrap>
+                  <FormField
+                    component={Input}
+                    name='opening_date'
+                    type='date'
+                    validate={isRequired}
+                  />
+                </DateInputWrap>
+              </Div>
+              <ColumnWrap>
+                <ProjectsCheckboxes />
+              </ColumnWrap>
+              <ColumnWrap>
+                { DAY_PARTS.map((day, index) => <DayPartSettings key={index} index={index} day={day}/>)}
+              </ColumnWrap>
+              <ButtonWrap>
+                { errorMessage && <ErrorMessage text={ errorMessage! } />}
+                { isUpdating && <SmallSpinner />}
+                <Button variant="secondary" disabled={hasValidationErrors}>Bewaren</Button>
+              </ButtonWrap>
+            </form>
+            <JSONDisplay json={ values } />
```

Comment:
I assume the displayed JSON now updates when the form changes. Could be a nice feature, but also maybe not.

Reply:
It does. Yes. 
Is that a problem? We could also display initialValues here.